### PR TITLE
Add in new "ci" subcommand

### PR DESCRIPTION
**Pull request by doanac**
This is primarily for adding two new subcommands:
 * ci workers
 * ci status

These give us a view like gavelci.net/status/ and ci.foundries.io/status/ have. They are hidden commands because only administrative users can run them.

While doing this work, I went ahead and moved the somewhat confusing "secrets" subcommand under this new CI umbrella to help make it more clear.

### CODE REVIEW

File: subcommands/ci/cmd.go [hunk 1]
```diff
@@ -8,14 +8,15 @@ import (
...
+}
+
 func NewCommand() *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "ci",
-		Short: "Commands that interact with the FoundriesFactory CI server",
-		PersistentPreRun: func(cmd *cobra.Command, args []string) {
-			api = subcommands.Login(cmd)
-		},
-	}
```
Comment (vkhoroz) [inline subcommands/ci/cmd.go]: I think it'd look nicer if it looks like this immediately in the `skeleton` commit.

File: client/foundries_ci.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+func (a *Api) CiWorkersList() ([]Worker, error) {
+	url := a.serverUrl + "/workers/"
+	type response struct {
+		Data struct {
+			Next    string   `json:"next"`
+			Workers []Worker `json:"workers"`
+		} `json:"data"`
+	}
```
Comment (vkhoroz) [inline client/foundries_ci.go:29]: I buy that you want this to be private, that's fine.
But, any reason it is not a module level type?

**Review by vkhoroz — APPROVED** [review]
LGTM

The only _serious_ comment is about deprecation.

File: client/foundries_ci.go (hunk 2 above)
Comment (doanac) [reply client/foundries_ci.go:29]: a big regret i have with jobserv was this "data" envelope. It serves no value or purpose. 

but - milo now implemented this support in admin.fio. I'm not sure this feature is actually needed any more in fioctl. I'm thinking we just close this for now and keep it in our back-pocket for another day.

**Pull request closed without merging**